packages/inscription-view/src/components/parts/common/role/MultipleRoleSelect.tsx

```diff
@@ -0,0 +1,36 @@
+import { useRoles } from './useRoles';
+import type { BrowserValue } from '../../../browser/Browser';
+import { Flex } from '@axonivy/ui-components';
+import { useBrowser } from '../../../browser/useBrowser';
+import { usePath } from '../../../../context/usePath';
+import Browser from '../../../browser/Browser';
+import Tags from '../../../widgets/tag/Tags';
+
+type MultipleRoleSelectProps = {
+  value: string[];
+  onChange: (change: string[]) => void;
+  onAdd: (change: string) => void;
+  showTaskRoles?: boolean;
+};
+
+const MultipleRoleSelect = ({ value, onChange, onAdd, showTaskRoles }: MultipleRoleSelectProps) => {
+  const { roles: roleItems } = useRoles(showTaskRoles);
+  const browser = useBrowser();
+  const path = usePath();
+  const selectedRoles = value ? value : ['Everybody'];
+
+  return (
+    <Flex direction='row' alignItems='center' gap={1} className='role-select'>
+      <Tags tags={selectedRoles} availableTags={roleItems.map(r => r.label)} customValues={false} onChange={change => onChange(change)} />
+      <Browser
+        {...browser}
+        types={['role']}
+        location={path}
+        accept={(change: BrowserValue) => onAdd(change.cursorValue)}
```

Comment:
Can't you use onChange here e.g. with
```suggestion
        accept={(change: BrowserValue) => onChange([...value, change.cursorValue])}
```